### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Fusselwurm**
* config example
* usage example
* link Wiki

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,30 @@
...
+```
+
+and execute/retrieve results like this:
+
+```sqf
+private _connection = dbCreateConnection "foo"; 
+private _query = dbPrepareQuery "foostmt"; 
+private _result = _connection dbExecute _query; 
+private _resultArray = dbResultToArray _result;
+systemChat format ["xxx %1", _resultArray];
```
Comment (Fusselwurm) [inline README.md]: up until this point, there are no errors - but formatting the `_resultArray` results in "array" , which is a bit weird  - I'd expect "[1]"  🤔

Comment (dedmen) [reply README.md]: "array" looks like it's nil.
Which means _result is nil? wat..

Comment (dedmen) [reply README.md]: So from top to bottom.
`createConnectionConfig` will return nil if the account doesn't exist in config...
That might be it, though your overall code is correct. I need to add a warning to that.

`dbPrepareQuery` can't return nil.
`dbExecute` can't return nil, but crashes game if query fails in unscheduled (already fixed on master)
`dbResultToArray` returns empty array if result wasn't correct.

So only cause I can see is createConnection not finding the account in config.

Comment (Fusselwurm) [reply README.md]: yes, please warn the shit out of us, or throw errors or … something. 
`nil` is the Value Of Satan , I never want to see it :D

Comment (Fusselwurm) [reply README.md]: I'll have a look at the config.yaml I used. :thinking:

**Pull request closed without merging**